### ISSUE TITLE
Inconsistent recording

### ISSUE THREAD

**Issue by nicxvan** [pducharme/UniFi-Video-Controller#185]
It seems that recording only works temporarily after restarting. I've tried constant recording as well as motion only.

I had to change the location of the recording since the volume I had was inaccessible.
I changed it to /var/lib/unifi-video/video

If I try to test the motion zone it does not seem to complete or provide any feedback.

```  unifivideo:
    image: pducharme/unifi-video-controller
    restart: always
    ports:
      - 10001:10001
      - 1935:1935
      - 6666:6666
      - 7080:7080
      - 7442:7442
      - 7443:7443
      - 7444:7444
      - 7445:7445
      - 7446:7446
      - 7447:7447
    cap_add:
      - DAC_READ_SEARCH
    tmpfs: /var/cache/unifi-video
    environment:
      - TZ="America/New_York"
      - PUID=99
      - PGID=100
      - CREATE_TMPFS=no
      - DEBUG=1
    volumes:
      - /mnt/surveillancevol1/unifidata:/var/lib/unifi-video
      - /mnt/surveillancevol1/unifivideos:/usr/lib/unifi-video/data/videos```

**Comment by thoschworks** [comment]
Looks like Issue https://github.com/pducharme/UniFi-Video-Controller/issues/178.

**Comment by nlighteneddesign** [comment]
I thought so at first, but I occasionally get recordings, it seems to only work after a restart before I make any settings changes.

**Comment by thoschworks** [comment]
I have problems like yours. But I didn't change the config for days and the recording stops suddenly. And it stops every time I reboot the host.

A few minutes ago I realized that it stopped again. (Perhaps cause I updated the system and got a new version of Docker? 🤔)

I looked into the container and `/var/cache/unifi-video/hls` had been vanished... (Same like in Issue https://github.com/pducharme/UniFi-Video-Controller/issues/178...)

(I should have look after the runtime of the container before I destroyed it...)
